src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/__init__.py

```diff
@@ -3,12 +3,25 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from azure.cli.core import AzCommandsLoader
+
 import azure.cli.command_modules.acr._help  # pylint: disable=unused-import
 
 
-def load_params(_):
-    import azure.cli.command_modules.acr._params  # pylint: disable=redefined-outer-name, unused-variable
+class ACRCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        super(ACRCommandsLoader, self).__init__(cli_ctx=cli_ctx,
+                                                min_profile='2017-03-10-profile')
```

Comment:
How will this profile version affect future development, such as which version we should set or update to?

Reply:
If at some point ACR is supported on Stack, this would change to the first supported Stack profile. Otherwise, you won't ever change this.